report/3.Theory/Dependability.tex

```diff
@@ -0,0 +1,45 @@
+\section{Dependability}
+Dependability in a real-time system is divided into two fields: hardware and software. 
+In the following subsections, the different methods and techniques used to achieve dependability will be elaborated upon.  
+In order to understand and achieve dependability, a fundamental understanding in regards to faults is needed, which will therefore be covered in this section.
+
+When dealing with dependability of a system, a key observation to be made is what defines the system. 
+\begin{quote}
+\textbf{System}: A system is an entity that interacts with other entities, i.e., other systems, including hardware, software, humans, and the physical world with its natural phenomena.
+These systems are the environment of the given system. 
+- John Knight \cite{FundementalsOfDependableComputingForSoftwareEngineers}
+\end{quote}
+It is said that a system is a system composed of systems, that are composed of systems and so on. 
+However, there is a system boundary between all the systems that composes the system and the system itself. 
+Hence dividing the system into a subject system and the systems in the environment. 
+This is important to notice when evaluating the dependability of the system. 
+It is inherent that a system, which is composed of an independent system, can never be more dependent than that system. 
```

Comment:
Is the more correct word not implicit? not inherent?

Reply:
^ I agree with this stuff